tests/unit/remote/test_index.py

```diff
@@ -0,0 +1,36 @@
+import pickle
+import os.path
+
+from dvc.remote.index import RemoteIndex
+from tests.basic_env import TestDvc
+
+
+class TestRemoteIndex(TestDvc):
```

Comment:
Let's use pytest-style tests. TestDvc is obsoleted and will be removed soon, we are fully migrating to pytest.